src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -467,6 +467,7 @@ class EmojiPickerMenu extends Component {
                     : (
                         <FlatList
                             ref={el => this.emojiList = el}
+                            initialNumToRender={400}
```

Comment:
This needs to be removed. This is not a solution to the problem you were mentioning. 
First, I do not understand the problem. Please add a video so that we can open this to broader audiences. 

This change will also hurt performance. 

I noticed a similar issue. When I was scrolling around the objects section of emoji, highlighted emoji was out of view. I think this is due to something else. Let's try maintainescrollPosition.

Reply:
scrollPosition is maintained `onScroll={e => this.currentScrollOffset = e.nativeEvent.contentOffset.y}`. But this value `this.currentScrollOffset` changes abruptly resulting the issue. I cannot find any bug in calculations in `scrollToHighlightedIndex()` function. Introduction of `initialNumToRender={100}` solving the issue also suggests no bugs in `scrollToHighlightedIndex()`

Reply:
> But this value this.currentScrollOffset changes abruptly resulting the issue.

Good catch, we are one step closer to the real problem now. Now the thing is to find out what is causing this .

Reply:
I cannot find the reason why `this.currentScrollOffset` value changes abruptly. I also cannot find the reason why, with the introduction of `initalNumToRender`, `this.currentScrollOffset` value does not change abruptly.

Reply:
Change made.